index.html

```diff
@@ -84,6 +84,254 @@ <h1 id="head-primary">
+
+          <div class="project-tags">
+            <ul>
+              <li class="html">HTML</li>
+              <li class="css">CSS</li>
+              <li class="javascript">JavaScript</li>
+            </ul>
+          </div>
+          <div class="project-action">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <!-- second card -->
+      <div class="project-card second-card">
+        <div class="snapshoot">
+          <img src="./src/assets/img/project-2.svg" alt="secondd project snapshoot">
+        </div>
+
+        <div class="left-block">
+
+          <div class="project-title">
+            <div class="title">
+              <p>Multi-Post <br> Stories</p>
+            </div>
+            <div class="frame">
+              <ul>
+                <li class="client">CANOPY</li>
+                <li><img src="./src/assets/img/counter.svg" alt=""></li>
+                <li class="role">Back End Dev</li>
+                <li><img src="./src/assets/img/counter.svg" alt=""></li>
+                <li class="year">2022</li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="project-info">
+            <p class="info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+
+          <div class="project-tags">
+            <ul>
+              <li class="html">HTML</li>
+              <li class="css">CSS</li>
+              <li class="javascript">JavaScript</li>
+            </ul>
+          </div>
+          <div class="project-action">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <!-- third card -->
+      <div class="project-card third-card">
+        <div class="snapshoot">
+          <img src="./src/assets/img/project-3.svg" alt="third project snapshoot">
+        </div>
+
+        <div class="left-block">
+
+          <div class="project-title">
+            <div class="title">
+              <p>Tonic</p>
+            </div>
+            <div class="frame">
+              <ul>
+                <li class="client">CANOPY</li>
+                <li><img src="./src/assets/img/counter.svg" alt=""></li>
+                <li class="role">Back End Dev</li>
+                <li><img src="./src/assets/img/counter.svg" alt=""></li>
+                <li class="year">2022</li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="project-info">
+            <p class="info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+
+          <div class="project-tags">
+            <ul>
+              <li class="html">HTML</li>
+              <li class="css">CSS</li>
+              <li class="javascript">JavaScript</li>
+            </ul>
+          </div>
+          <div class="project-action">
+            <button type="button">See Project</button>
```

Comment:
- [ ] [REQUIRED] Good job you made some changes requested by the previous reviewer [here](https://github.com/iAmWaris97/iAmWaris97-Portfolio/pull/2#discussion_r917302638) :+1:   but you need to add the `white` color to all your button background when is `enabled` as defined in the Figma template because it very important that you stick the design as much as possible :smile: 

### Figma button enable design 
![Screenshot from 2022-07-09 20-54-13](https://user-images.githubusercontent.com/6615394/178122483-341f77b8-861f-40aa-a9ad-1f5dcaf57413.png)

### Your project
![Screenshot from 2022-07-09 20-53-32](https://user-images.githubusercontent.com/6615394/178122486-215a84a8-ee1a-46d0-94e3-8f0a82eec72e.png)